modules/data/cosmos-db/README.md

```diff
@@ -0,0 +1,99 @@
+# Azure Cosmos DB
+
+Bicep module for simplified deployment of Cosmos DB; enables VNet integration and offers flexible configuration options.
+
+## Description
+
+This Bicep Module simplifies the deployment of a Cosmos DB account by providing a reusable set of parameters and resources.
+It allows for the creation of a new Cosmos DB account or use of an existing one, and offers configuration options such as the default consistency level, system-managed failover, and multi-region writes.
+The module also allows for the attachment of the Cosmos DB account to a virtual network, if specified.
```

Comment:
Is the statument about vnet attachment true? I didn't find the bicep code regarding that.

Reply:
It looks like I lost this in my copying from repo to repo.... let me inspect

Reply:
```suggestion
```